### PR TITLE
Readme directs to temp repo

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@danroth27 please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,5 +3,9 @@ ASP.NET Docs
 
 This project provides the source for [docs.asp.net](http://docs.asp.net/). You can learn more about ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
 
-**We accept pull requests!** However, before submitting a pull request, please [read the CONTRIBUTING guidelines](CONTRIBUTING.md), which include information on how to build the docs locally, as well as style and organizational guidance.
+~~**We accept pull requests!** However, before submitting a pull request, please [read the CONTRIBUTING guidelines](CONTRIBUTING.md), which include information on how to build the docs locally, as well as style and organizational guidance.~~
+
+Please make pull requests at our [temporary repo](https://github.com/Microsoft/Docs-1). 
+
+API PR's ( *api/autoapi/...* ) should not be made here. Changes to API documentation should be made in the source repro against the tripple slash /// comments.
```
Comment (danroth27) [inline README.md]: triple -> triple

File: README.md [hunk 2]
```diff
@@ -3,5 +3,9 @@ ASP.NET Docs
 
 This project provides the source for [docs.asp.net](http://docs.asp.net/). You can learn more about ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
 
-**We accept pull requests!** However, before submitting a pull request, please [read the CONTRIBUTING guidelines](CONTRIBUTING.md), which include information on how to build the docs locally, as well as style and organizational guidance.
+~~**We accept pull requests!** However, before submitting a pull request, please [read the CONTRIBUTING guidelines](CONTRIBUTING.md), which include information on how to build the docs locally, as well as style and organizational guidance.~~
```
Comment (danroth27) [inline README.md]: I wouldn't leave this crossed out section. It looks like we changed our mind on accepting community contributions, which we haven't.

File: README.md [hunk 3]
```diff
@@ -3,5 +3,9 @@ ASP.NET Docs
...
+
+Please make pull requests at our [temporary repo](https://github.com/Microsoft/Docs-1).
```
Comment (danroth27) [inline README.md]: I don't think we are actually setup to accept contributions on the Docs-1 repo. That repo doesn't have a license specified and I'm guessing it also doesn't have the CLA bot setup. I think we should just say that we are moving to [DocFX](http://dotnet.github.io/docfx/) and that we are temporarily holding off on accepting contributions until the migration is complete (give an ETA).

File: README.md (hunk 1 above)
Comment (tdykstra) [inline README.md]: Also repro -> repo

**Pull request closed without merging**